site/content/docs/5.3/getting-started/parcel.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: docs
-title: "Boosted & Parcel"
+title: Boosted and Parcel
 description: The official guide for how to include and bundle Boosted's CSS and JavaScript in your project using Parcel.
 group: getting-started
 aliases:
   - "/docs/getting-started/parcel/"
 toc: true
-thumbnail: parcel.png
+thumbnail: guides/parcel-dev-server-boosted.png
```

Comment:
It's not possible to leave it as it was because the images are not all with the PNG format.
We agreed with the designers that we don't want to lose time to create specific images for Boosted.
So the thing I can do is to let the default image instead of these ones.